set-quotas.py

```diff
@@ -122,17 +122,19 @@ def match_keystone_project(all_ks_projects, form_project):
     config.read('settings.ini')
     admin_user = config.get('auth', 'admin_user')
     admin_pwd = config.get('auth', 'admin_pwd')
-    admin_tenant = config.get('auth', 'admin_tenant')
+    admin_project = config.get('auth', 'admin_project')
+    auth_url = config.get('auth', 'auth_url')
     nova_version = config.get('nova', 'version')
-    keystone_v3_url = config.get('setpass', 'keystone_v3_url')
     quota_auth_file = config.get('quota_sheet', 'auth_file')
     quota_worksheet_key = config.get('quota_sheet', 'worksheet_key')
     quota_template = config.get('quota_email', 'template')
 
     # openstack auth
-    auth = v3.Password(auth_url=keystone_v3_url,
+    auth = v3.Password(auth_url=auth_url,
```

Comment:
Yes, that's what I wanted to ask. Previously, I was working on upgrading to mitaka in my staging and faced this issue. I was looking through my bookmarks and found this:

http://lists.openstack.org/pipermail/openstack-dev/2016-March/088636.html

Also, let me search again a little more on this and get back to you. Did you check with Kristi?